it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/monitoring/MonitoringClient.java

```diff
@@ -410,6 +410,53 @@ public List<Long> listTimeSeriesAsLong(ListTimeSeriesRequest request) {
     return Collections.max(timeSeries);
   }
 
+  /**
+   * Gets aggregated metric for a given filter string for the given interval. The aggregation will
+   * be applied over the entire interval and only one value will be returned.
+   *
+   * @param projectId the project id
+   * @param filter filter string
+   * @param interval time interval
+   * @param aggregationFunction the aggregation function to apply on the time series for the whole
+   *     interval
+   */
+  public @Nullable Double getAggregatedMetric(
```

Comment:
Does this package have unit tests? If yes, can we add for these as well?

Reply:
MonitoringClient doesn't have unit tests. I'll check if I can add.